var/spack/repos/builtin/packages/tcl/package.py

```diff
@@ -25,7 +25,7 @@ class Tcl(AutotoolsPackage):
     version('8.6.5', '0e6426a4ca9401825fbc6ecf3d89a326')
     version('8.6.4', 'd7cbb91f1ded1919370a30edd1534304')
     version('8.6.3', 'db382feca91754b7f93da16dc4cdad1f')
-    version('8.5.19', '0e6426a4ca9401825fbc6ecf3d89a326')
+    version('8.5.19', '4f4e1c919f6a6dbb37e9a12d429769a6')
```

Comment:
Any idea why this hash changed?

Reply:
I'm not sure. Is this something that needs to be investigated?

Reply:
Probably if we want to avoid another https://github.com/Homebrew/homebrew-cask/issues/33536

Reply:
The md5 sum matches what is listed in https://sourceforge.net/projects/tcl/files/Tcl/8.5.19/ and the tarball is dated Feb 9, 2016. I'm not sure where the initial md5 sum in spack came from. Is there something else that can be done for validation?

Reply:
It looks like @citibeth added 8.5.19 in #716. @citibeth you don't happen to still have that tarball lying around do you? I'm not sure what the procedure is in this kind of situation.

Reply:
@citibeth please let us know if you have any insights here. @adamjstewart do you consider this a show stopper? I don't know if there is general guidance for this or if there has been wider discussion of changing checksums. @tgamblin @becker33 ?

Reply:
I usually consider it a showstopper. I'm not sure how to resolve this. I usually contact the developers and see if they know why the checksum changed. Usually they just added a patch without incrementing the version.

Reply:
I rooted around for copies of the tarball.  In all cases, the tarball I downloaded checksummed to `4f4...`:
    https://packages.ubuntu.com/source/xenial/tcl8.5
    http://ftp.pdp-11.org.ru/OpenBSD/distfiles/

I don't know how or why I added version `8.5.19`, or even if I ever installed it (probably not).  I would chalk it up to an error on my part.  I think it's best to check the other version hashes that were added in #716.